backend/test/integration/run_tests_locally.sh

```diff
@@ -32,7 +32,22 @@ case "$response" in
         ;;
 esac
 
-echo "Starting integration tests..."
-command="go test -v ./... -namespace ${NAMESPACE} -args -runIntegrationTests=true -isDevMode=true"
-echo $command "$@"
-$command "$@"
+if [ "$1" == "postgres" ]; then
+    echo "Starting PostgreSQL DB port forwarding..."
+    kubectl -n "$NAMESPACE" port-forward svc/postgres-service 5432:5432 --address="127.0.0.3" & PORT_FORWARD_PID=$!
```

Comment:
What is this part used for ? `PORT_FORWARD_PID=$!`

Reply:
Thank you for catching this! I should have added the function that cleans up the process on exit.

Reply:
added